Gopkg.toml

```diff
@@ -9,6 +9,8 @@ required = [
   "k8s.io/code-generator/cmd/lister-gen",
   "k8s.io/code-generator/cmd/informer-gen",
   "github.com/knative/pkg/codegen/cmd/injection-gen",
+  # TODO(#1996): Drop this when we drop our patches.
```

Comment:
You have this and a `Fixes:` in the PR body.  Can you open an issue to clean this up with an indication of which K8s version that might be?

Reply:
Filed https://github.com/knative/serving/issues/4549 and rebased.